windowPreview.js

```diff
@@ -95,7 +95,7 @@ const WindowPreviewList = new Lang.Class({
     Name: 'WindowPreviewMenuSection',
     Extends: PopupMenu.PopupMenuSection,
 
-    _init: function(app, settings) {
```

Comment:
The reason is we can use the `source`  in line 209 for 
```
let newWin = this._source.getInterestingWindows().sort(this.sortWindowsCompareFunction);
```
At the moment `this._source` is undefined.

The alternative is passing the necessary arguments to call the global function `getInterestingWindows`.